Program.cs

```diff
@@ -1,7 +1,39 @@
 ﻿namespace Training {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello World!");
+         Console.WriteLine ("Enter a password:");
+         string password = Console.ReadLine ();
+
+
+         bool length = password.Length >= 6;
+         bool digit = password.Any (char.IsDigit);
+         bool lower = password.Any (char.IsLower);
+         bool upper = password.Any (char.IsUpper);
+         bool specialchar = password.Any (c => "!@#$%^&*()-+".Contains (c));
+         bool var = length && digit && lower && upper && specialchar;
+
+
+         if (var == true) {

```

Comment:
```suggestion
         if (length && digit && lower && upper && specialchar) {
```

This way local variable var can be removed. Also var is a terrible name for a local variable in c#